sencha-workspace/SlateDemonstrationsStudent/app/controller/Dashboard.js

```diff
@@ -222,17 +219,26 @@ Ext.define('SlateDemonstrationsStudent.controller.Dashboard', {
             }
         }
 
+        // get an array of competency IDs from the studentCompetenciesStore
+        let studentCompetencyIDs = studentCompetenciesStore.getData().items.map(rec => rec.get('CompetencyID'));
```

Comment:
**suggestion (non-blocking)**: I think you can just do this. Also note preferring `const` when a variable doesn't need to be reassigned

```suggestion
        const studentCompetencyIDs = studentCompetenciesStore.collect('CompetencyID');
```